src/Types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export type Todo = {
```

Comment:
better to create interface
```suggestion
export interface Todo {
```